galaxy/tools/deps/mulled/get_tests.py

```diff
@@ -0,0 +1,263 @@
+    except IOError: 
+        logging.info('/home/ubuntu/GitRepos/bioconda-recipes/recipes/%s/%s/meta.yaml could not be opened.' % (name[0], name[1]))
+        t = None
+    if t:
+        t['container'] == container
+        return t
+
+    # try run_test in base folder
+
+    try:
+        t = get_runtest(open_recipe_file('recipes/%s/%s/run_test.sh' % (name[0], name[1])))
+    except IOError: 
+        logging.info('/home/ubuntu/GitRepos/bioconda-recipes/recipes/%s/%s/run_test.sh could not be opened.' % (name[0], name[1]))
+        t = None
+    if t:
+        t['container'] == container
+        return t
+
+    # now try meta.yaml in base folder
+    try:
+        t = get_commands_from_yaml(open_recipe_file('recipes/%s/meta.yaml' % name[0]))
+    except IOError: 
+        logging.info('/home/ubuntu/GitRepos/bioconda-recipes/recipes/%s/meta.yaml could not be opened.' % name[0])
+        t = None
+    if t:
+        t['container'] == container
+        return t
+
+    # try run_test in base folder
+
+    try:
+        t = get_runtest(open_recipe_file('recipes/%s/run_test.sh' % name[0]))
+    except IOError: 
+        logging.info('/home/ubuntu/GitRepos/bioconda-recipes/recipes/%s/run_test.sh could not be opened.' % name[0])
+        t = None
+    if t:
+        t['container'] == container
+        return t
+
+    # try from anaconda
+
+    t = get_test_from_anaconda(get_anaconda_url(container))
+    if t:
+        t['container'] == container
+        return t
+    logging.info('Nothing on anaconda.')
+    # now try in incorrect version folders
+
+    #g = glob('/home/ubuntu/GitRepos/bioconda-recipes/recipes/%s/*/meta.yaml' % container)
+    g = get_alternative_versions('recipes/%s' % container, 'meta.yaml')
+    for n in g:
+        try:
+            t = get_commands_from_yaml(open(n))
+        except IOError:
+            logging.info('No wrong versions (meta.yaml) either in recipes repo.')
+            t = None
+        if t:
+            t['container'] == container
+            return t
+
+    # g = glob('/home/ubuntu/GitRepos/bioconda-recipes/recipes/%s/*/run_test.sh')
+    g = get_alternative_versions('recipes/%s' % container, 'run_test.sh')
+    for n in g:
+        try:
+            t = get_commands_from_yaml(open(n))
+        except IOError:
+            logging.info('No wrong versions (run_test.sh) either in recipes repo.')
+            t = None
+        if t:
+            t['container'] == container
+            return t
+
+    g = find_anaconda_versions(container)
+    for n in g:
+        t = get_test_from_anaconda(n)
+        if t:
+            t['container'] == container
+            return t
+    logging.info('And no wrong versions in anaconda.')
+            
+    return {}
+
+######################################
+########## MAIN STARTS HERE ##########
+######################################
+
+RECIPES_REPO_PATH = '/home/simon/GitRepos/bioconda-recipes' # can also be None; then the the repository will be accessed via github
```

Comment:
Here is something hardcoded, this should be a parameter that the user can optionally pass into the script.